packages/dashboard/src/app/views/myStories/content/index.js

```diff
@@ -45,10 +45,29 @@ import {
 import { EmptyContentMessage } from '../../shared';
 import StoriesView from './storiesView';
 
+function NoAvailableContent({ keyword, filtersObject }) {
+  if (keyword) {
+    return sprintf(
+      /* translators: %s: search term. */
+      __('Sorry, we couldn\'t find any results matching "%s"', 'web-stories'),
+      keyword
+    );
+  } else if (Object.keys(filtersObject).length !== 0) {
+    return __("Sorry, we couldn't find any results", 'web-stories');
+  } else {
+    return __('Start telling Stories.', 'web-stories');
+  }
+}
+NoAvailableContent.propTypes = {
+  filtersObject: PropTypes.object,
+  keyword: PropTypes.string,
+};
+
 function Content({
   allPagesFetched,
   canViewDefaultTemplates,
   filter,
+  filtersObject = {},
```

Comment:
🤔 You could pull this straight from the `FiltersProvider` instead of passing props - but not totally necessary

Reply:
Let me see, that could be easy to do